schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     DATE_OF_BIRTH DATE, 
     ESCAPE_ATTEMPTS INT, 
     NEUTERED BOOLEAN, 
-    WEIGHT_KG REAL
+    WEIGHT_KG REAL,
+    SPECIES VARCHAR(60)
```

Comment:
- [ ] I think this is not the right way to add a colmun you should use the alter command so you can add a column to an existed table so kindly use the alter command for adding a new column. ✅🙏🏻

You can add somthing like this
> ALTER TABLE animals ADD species varchar(255);